src/main/kotlin/io/github/rybalkinsd/kohttp/util/json.kt

```diff
@@ -41,7 +41,7 @@ class Json {
     }
 
     override fun toString(): String =
-        elements.joinToString(separator = ",", prefix = "{", postfix = "}") {
-            (k,v) -> """"$k":$v"""
```

Comment:
what is the reason to change this fragment?

Reply:
Lambda formatting - https://kotlinlang.org/docs/reference/coding-conventions.html#lambda-formatting
Tabulation i will change back